src/DestinationCard/index.js.flow

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import type { Globals, Translation } from "../common/common.js.flow";
+
+export type State = {|
+  hiddenContentHeight: number,
+|};
+
+export type Props = {|
+  ...Globals,
+  +departureCity: Translation,
+  +destinationCity: string,
```

Comment:
This one also 🙏

Reply:
It conflitcs with Lazyimage types, it's passed to it. Should in this case also update the Lazyimage types?

Reply:
Yeah, sure. Even that we use `Translation` type, devs should pass `() => string`.

Reply:
OK, on it!